src/sentry/issues/grouptype.py

```diff
@@ -338,10 +338,10 @@ class PerformanceDurationRegressionGroupType(PerformanceGroupTypeDefaults, Group
 
 
 @dataclass(frozen=True)
-class PerformanceP95DurationRegressionGroupType(PerformanceGroupTypeDefaults, GroupType):
+class PerformanceP95EndpointRegressionGroupType(PerformanceGroupTypeDefaults, GroupType):
     type_id = 1018
-    slug = "performance_p95_duration_regression"
-    description = "Transaction Duration Regression"
+    slug = "performance_p95_endpoint_regression"
```

Comment:
I don't know if `post-process` and `ingest` options would get renamed to match the new slug. You might need to create a new group type. I'd confirm with @wedamija how slug renaming is handled

Reply:
Also you'll need to update the feature handler in [getsentry](https://github.com/getsentry/getsentry/blob/master/getsentry/features.py#L1140) and it might cause issues with options-automator as well because I had issues when I renamed a feature handler previously

Reply:
From what I can tell, the slug rename should transition to the new slug smoothly. But good call out on the feature handler, I don't think that should exist because there's a default one that can be used for the rollout provided by the group type. Will take a look and see what next steps are.

Reply:
So as far as I can tell, the slug rename is safe as the issue itself is currently unused so it'll just unregister the old flags + options then register the new flags and options.

And as for the feature handler in getsentry, it was using some unregistered feature flags. i.e. it was never in use so I've gone ahead and removed all of them.

Reply:
sounds good 👍